pom.xml

```diff
@@ -160,35 +160,6 @@
     <build>
         <pluginManagement>
             <plugins>
-                <!-- this plugin is a maven plugin designed to hold meta data to communicate with eclipse m2e plugin when 
-                    it processes a maven pom.xml. Without this, children of this pom file will get the warning 
-                    "maven-enforcer-plugin (goal enforce) is ignored by m2e" because there is no connector that tells 
-                    m2e how to integrate the maven plugin into eclipse. -->
-                <plugin>
```

Comment:
Recent m2e versions don't need this anymore and I don't believe any recent version of eclipse *requires* this now. I know for certain eclipse (neon) builds SeLion fine w/o this.  For users still on old versions of Eclipse (for some reason) the recommendation is to add this to ~/.m2/settings.xml